test/cloud-server/Dockerfile

```diff
@@ -7,15 +7,20 @@ COPY . .
 RUN ( cd /usr/local/go && patch -p1 < $GOPATH/src/github.com/plgd-dev/hub/tools/docker/patches/shrink_tls_conn.patch )
 ARG root_directory=$GOPATH/src/github.com/plgd-dev/hub
 
+#certificate-generator
+ARG service=kit
+WORKDIR /
+RUN cd $GOPATH/pkg/mod/github.com/plgd-dev/kit/v2* && go build -ldflags "-linkmode external -extldflags -static" -o /go/bin/certificate-generator ./cmd/certificate-generator
```

Comment:
use `tools/cert-tool/cmd`

Reply:
updated in Dockerfile.test, bundle/Dockerfile and test/cloud-server/Dockerfile